src/client/components/Post/index.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import './style.scss';
+import PostHeader from '../PostHeader/';
+import PostInfo from '../PostInfo/';
+import PostAct from '../PostAct/';
+
+class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.PostInfo = {
```

Comment:
please add a default value to the props instead of using a new object. Now if we will introduce the ajax communication we will have to change all of the references to the PostInfo